docs/concepts/component_architecture/flytepropeller_architecture.rst

```diff
@@ -30,12 +37,99 @@ This document attempts to break down the FlytePropeller architecture by tracking
 Components
 ==========
 
+
+FlyteAdmin is the common entry point, where initialization of FlyteWorkflow CRs may be triggered by user workflow definition executions, automatic relaunches, or periodically scheduled workflow definition executions.
+
 FlyteWorkflow CRD / K8s Integration
 -----------------------------------
 
-Workflows in Flyte are maintained as Custom Resource Definitions (CRDs) in Kubernetes, which are stored in the backing etcd cluster. Each execution of a workflow definition results in the creation of a new FlyteWorkflow CR (Custom Resource) which maintains a state for the entirety of processing. CRDs provide variable definitions to describe both resource specifications (spec) and status (status). The FlyteWorkflow CRD uses the spec subsection to detail the workflow DAG, embodying node dependencies, etc. The status subsection tracks workflow metadata including overall workflow status, node/task phases, status/phase transition timestamps, etc.
+Workflows in Flyte are maintained as `Custom Resource Definitions (CRDs) <https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/>`__  in Kubernetes, which are stored in the backing ``etcd`` key-value store. Each workflow execution results in the creation of a new ``flyteworkflow`` CR (Custom Resource) which maintains its state for the duration of the execution. CRDs provide variable definitions to describe both resource specifications (``spec``) and status (``status``). The ``flyteworkflow`` CRD uses the ``spec`` subsection to detail the workflow DAG, embodying node dependencies, etc. 
+
+**Example**
+
+1. Execute an `example workflow <https://docs.flyte.org/en/latest/core_use_cases/machine_learning.html#machine-learning>`__ on a remote Flyte cluster:
+
+.. code-block:: bash 
+
+   pyflyte run --remote example.py training_workflow --hyperparameters '{"C": 0.4}'
+
+2. Verify there's a new Custom Resource on the ``flytesnacks-development`` namespace (this is, the workflow belongs to the ``flytesnacks`` project and the ``development`` domain):
+
+.. code-block:: bash 
+
+   kubectl get flyteworkflows.flyte.lyft.com -n flytesnacks-development
+
+Example output:
+
+.. code-block:: bash 
+
+   NAME                   AGE
+   f7616dc75400f43e6920   3h42m 
+
+3. Describe the contents of the CR, for example the ``spec`` section:
+
+.. code-block:: bash 
+
+   kubectl describe flyteworkflows.flyte.lyft.com f7616dc75400f43e6920 -n flytesnacks-development  
+
+.. code-block:: json 
+
+    "spec": {
+        "connections": {
+            "n0": [
+                "n1"
+            ],
+            "n1": [
+                "n2"
+            ],
+            "n2": [
+                "end-node"
+            ],
+            "start-node": [
+                "n0",
+                "n2"
+            ]
+        },
+
+The status subsection tracks workflow metadata including overall workflow status, node/task phases, status/phase transition timestamps, etc.
```

Comment:
I'm not sure why the rest of this doc is highlighted red starting at this line -- is there a formatting issue here?

Reply:
Not sure either. I see the rst renders correctly and I compared the JSON blocks to what the multicluster doc has and it follows the same structure so not sure.

Reply:
Gotcha -- as long as the rst renders correctly, it should be fine.